polyfills/Intl/update.task.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var rimraf = require('rimraf');
+var IntlSourcePath = require.resolve('intl/dist/Intl.min.js');
+var LocalesPath = path.dirname(require.resolve('intl/locale-data/jsonp/en.js'));
+var IntlPolyfillOutput = path.resolve('polyfills/Intl');
+var LocalesPolyfillOutput = path.resolve('polyfills/Intl/~locale');
+
+// this is not really a grunt task, but a function that is suppose
+// to be sync, and receive access to grunt instance for convenience.
```

Comment:
This is the part that I don't like, but I could find a reliable way to make update.task.js a real grunt task that can be loaded and executed dynamically.